packages/itwin/ec3-widget/src/components/TemplateMenu.scss

```diff
@@ -2,53 +2,36 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-
-.ec3w-template-details-container {
-  overflow-x: auto;
+.ec3-report-creation-stepper {
```

Comment:
Inconsistent prefix - use `ec3w` instead of `ec3`.

Reply:
done